src/test/java/EventTest.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.Test;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class EventTest {
+    @Test
+    public void testEventToString(){
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        Event event = new Event("meeting", LocalDateTime.parse("2000-01-01 2359", formatter));
+        assertEquals("[E][\u2718]meeting(at: Jan 1 2000 23:59)", event.toString());
+    }
+
+    @Test
+    public void testSetDone(){
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+        Event event = new Event("return book", LocalDateTime.parse("2000-01-01 2359", formatter));
+        assertEquals(false, event.getStatus());
```

Comment:
You can use `assert(!event.getStatus());` here as well, just to make the code slightly cleaner, similar comment for line 20. The rest looks good, great job!